### PR TITLE
Add Create New Event page

### PR DESCRIPTION
**Pull request by Scott-Borecki**
### Changes Implemented:
  - Created New Event page and functionality

###  Quality Control Checklist:
  - [ ] Code adheres to Rubocop styling (if unavoidable infractions, please clarify below)
  - [ ] 100% SimpleCov test coverage (if below, please clarify below)
    - Coverage is slipping with time not on our side.
  - [X] Last commit has passing Travis build

### Next Steps & Additional Notes:
  - Need to connect gym show page to the create new event page
  - Need to test the facade/service in production to confirm the event is being created

### CODE REVIEW

File: app/controllers/events_controller.rb [hunk 1]
```diff
@@ -1,5 +1,21 @@
...
+  end
+
+  def create
+    new_event = BackEndFacade.create_event(create_event_params)
+    if new_event.present?
+      flash[:alert] = 'Workout created! Now go get your swell on!'
+      redirect_to dashboard_path(current_user.id)
+    else
+      flash[:alert] = 'Error: Something went wrong!'
+      redirect_to URI(request.referer).path
```
Comment (tvaroglu) [inline app/controllers/events_controller.rb:16]: Gonna have to circle back to wrap my head on this syntax when more time

File: app/controllers/events_controller.rb [hunk 2]
```diff
@@ -16,4 +32,20 @@ def event_destroy_params
...
   end
+
+  def new_event_params
+    params.permit(:friend_id, :friend_name, :gym_membership_id, :gym_name)
+  end
+
+  def create_event_params
+    params[:date_time] = DateTime.civil(
+      params[:date]['when(1i)'].to_i,
+      params[:date]['when(2i)'].to_i,
```
Comment (tvaroglu) [inline app/controllers/events_controller.rb:43]: Clutch helper method!!

File: app/controllers/events_controller.rb (hunk 1 above)
Comment (Scott-Borecki) [reply app/controllers/events_controller.rb:16]: This just sends the user back to the place they came from (in this case, the new event page).

File: app/services/back_end_service.rb [hunk 3]
```diff
@@ -33,6 +33,13 @@ def create_user(user_params)
       )
     end
 
+    def create_event(params)
+      db_conn.post("/api/v1/users/#{params[:user_id]}/gym_memberships/#{params[:gym_membership_id]}/events") do |req|
+        req.params['activity'] = params[:activity]
+        req.params['date_time'] = params[:date_time]
```
Comment (tvaroglu) [inline app/services/back_end_service.rb:39]: Idk if this is right... my recommendation would be to follow the only syntax we could get working for `create_user` (in prod)

**Review by tvaroglu — APPROVED** [review]
Nice work!!  Approved

**Pull request merged**